examples/fluids/index.md

```diff
@@ -432,6 +432,9 @@ The simulation produces a strong acoustic wave and leaves behind a cold thermal
 The boundary conditions are freestream in the x and y directions. When using an HLL (Harten, Lax, van Leer) Riemann solver {cite}`toro2009` (option `-freestream_riemann hll`), the acoustic waves exit the domain cleanly, but when the thermal bubble reaches the boundary, it produces strong thermal oscillations that become acoustic waves reflecting into the domain.
 This problem can be fixed using a more sophisticated Riemann solver such as HLLC {cite}`toro2009` (option `-freestream_riemann hllc`, which is default), which is a linear constant-pressure wave that transports temperature and transverse momentum at the fluid velocity.
 
+## Vortex Shedding - Flow past Cylinder
+This test case is taken from {cite}`shakib1991femcfd`. At time $t=0$, we introduce a circular cylinder with diameter $D$ centered at $x = y = 0$ in a uniform flow. The viscosity is assumed constant. We use the computational domain $-4.5 \leq x \leq 15.5$, $-4.5 \leq y \leq 4.5$. On the inflow boundary $(x = -4.5)$, pressure, velocity and temperature are prescribed. On the top and bottom boundaries $(y = \pm 4.5)$, symmetry condition, zero normal velocity component, zero heat-flux are imposed. On the outflow boundary $(x = 15.5)$, zero traction is prescribed. On the cylinder wall, a no-slip boundary condition, no heat-flux condition are imposed. Boundary conditions are freestream at inflow and outflow boundaries. As we evolve in time, eddies appear past the cylinder leading to a vortex shedding which is known as the vortex street.
```

Comment:
1. What is the initial condition? It is not the same as the freestream condition.
2. You have described the inflow and outflow conditions incorrectly at length (they're physically invalid subsonic inflow and outflow conditions, but we also aren't using what the code describes as `inflow` and `outflow`) and then contradict what you just wrote by saying they are "freestream" (correct!).
3. State the Reynolds number and Mach number as currently configured.